bin/check-aggregate.rb

```diff
@@ -185,6 +185,32 @@ def acquire_aggregate
   def named_aggregate_results
     results = api_request("/aggregates/#{config[:check]}?max_age=#{config[:age]}")[:results]
     warning "No aggregates found in last #{config[:age]} seconds" if %w(ok warning critical unknown).all? { |x| results[x.to_sym] == 0 }
+
+    err_results = []
+    if results[:critical] > 0
```

Comment:
We should use unknown in the case of a negative number (I know not likely but false positives often seem to these silly little error handling)